use_cases/eluc/prescriptors/prescriptor_manager.py

```diff
@@ -0,0 +1,49 @@
+"""
+Manages multiple Prescriptors and a Predictor. Allows the user to easily prescribe based on different Prescriptors and
+then predict on a uniform Predictor in order to compare them.
+Additionally handles the percent changed computation.
+"""
+import pandas as pd
+
+from data import constants
+from predictors.predictor import Predictor
+from prescriptors.prescriptor import Prescriptor
+
+class PrescriptorManager():
+    """
+    Stores many Prescriptor objects and a predictor.
+    Used to uniformly prescribe and predict various Predictors.
+    """
+    def __init__(self, prescriptors: dict[str, Prescriptor], predictor: Predictor):
+        self.prescriptors = prescriptors
+        self.predictor = predictor
+
+    def prescribe(self, cand_id: str, context_df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Prescribes from a context using a specific candidate.
+        """
+        return self.prescriptors[cand_id].prescribe(context_df)
+
+    def predict_metrics(self, context_actions_df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Computes ELUC and change for each sample in a context_actions_df.
+        """
+        eluc_df = self.predictor.predict(context_actions_df)
+        change_df = self.compute_percent_changed(context_actions_df)
+
+        return eluc_df, change_df
+
+    # TODO: Move this to its own predictor
```

Comment:
Right, `compute_percent_changed` could be a `Predictor`. Means the constructor of `PrescriptorManager` should probably take a list of Predictor objects instead of a single one. And to compute the metrics we loop over the predictors. For another PR.